### PR TITLE
legacy links updated (tw 219)

### PR DESCRIPTION
**Pull request by DJSanti**
## What was changed
Links across the repo and to external pages were updated.

## Why?
Recent documentation changes mean that some pages are under new directories or no longer exist.

### CODE REVIEW

**Review by bergundy — APPROVED** [review]

File: activities-cancellation-heartbeating/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Running [`src/client.ts`](./src/client.ts) does this:
 
 ### Running this sample
 
-1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/server/quick-install/)).
+1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/clusters/quick-install)).
```
Comment (flossypurse) [inline activities-cancellation-heartbeating/README.md:27]:
```suggestion
1. Make sure Temporal Server is running locally (see the [Dev guide](https://docs.temporal.io/application-development/foundations#run-a-development-cluster)).
```

File: activities-dependency-injection/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-This sample shows how to [share dependencies](https://docs.temporal.io/typescript/activities#sharing-dependencies-in-activity-functions) between Activities: for example, when you need to initialize a database connection once and then pass it to multiple Activities.
+This sample shows how to [share dependencies](https://docs.temporal.io/application-development/foundations?lang=typescript/#develop-activities) between Activities: for example, when you need to initialize a database connection once and then pass it to multiple Activities.
 
 [`src/worker.ts`](./src/worker.ts)
 
 ### Running this sample
 
-1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/server/quick-install/)).
+1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/clusters/quick-install)).
```
Comment (flossypurse) [inline activities-dependency-injection/README.md:9]:
```suggestion
1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/application-development/foundations#run-a-development-cluster)).
```

File: activities-examples/README.md [hunk 3]
```diff
@@ -3,8 +3,8 @@
 This sample shows use cases for Activities:
 
 - [`makeHTTPRequest`](./src/activities/index.ts): Make an external HTTP request in an Activity (using `axios`).
-- [`cancellableFetch`](./src/activities/cancellable-fetch.ts): Make a cancellable HTTP request with [cancellationSignal](https://typescript.temporal.io/api/classes/activity.context/#cancellationsignal).
-- [`doSomethingAsync`](./src/activities/async-completion.ts): Complete an Activity async with [`AsyncCompletionClient`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient#complete).
+- [`cancellableFetch`](./src/activities/cancellable-fetch.ts): Make a cancellable HTTP request with [cancellationSignal](https://docs.temporal.io/application-development/testing/#cancel-an-activity).
```
Comment (flossypurse) [inline activities-examples/README.md:6]: I don't think you want to change this one?

File: activities-examples/README.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ More Activity samples are [listed here](https://github.com/temporalio/samples-ty
 
 ### Running this sample
 
-1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/server/quick-install/)).
+1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/clusters/quick-install)).
```
Comment (flossypurse) [inline activities-examples/README.md:20]:
```suggestion
1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/application-development/foundations#run-a-development-cluster)).
```

File: activities-sticky-queues/README.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ Activities have been artificially slowed with `activity.Context().sleep(3000)` t
 
 ### Running this sample
 
-1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/server/quick-install/)).
+1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/clusters/quick-install)).
```
Comment (flossypurse) [inline activities-sticky-queues/README.md:17]:
```suggestion
1. Make sure Temporal Server is running locally (see the [quick install guide](https://docs.temporal.io/application-development/foundations#run-a-development-cluster)).
```

**Review by flossypurse — CHANGES_REQUESTED** [review]

File: activities-examples/README.md (hunk 3 above)
Comment (DJSanti) [reply activities-examples/README.md:6]: didn't know the link vanished. thanks

Comment (flossypurse) [reply activities-examples/README.md:6]: not sure what you mean.
But https://typescript.temporal.io/api/classes/activity.context/#cancellationsignal should not change to https://docs.temporal.io/application-development/testing/#cancel-an-activity
So this needs to be reverted.

Comment (DJSanti) [reply activities-examples/README.md:6]: Page wasn't found. May have taken on an additional directory, so I'll check on that

**Review by vkarpov15 — APPROVED** [review]
:+1:

**Review by flossypurse — APPROVED** [review]

**Comment by DJSanti** [comment]
Is this okay to merge despite the failing check on lint?

**Comment by vkarpov15** [comment]
@DJSanti I pushed 3c97c7a which should fix the lint issue. Sorry about the slightly persnickety lint script.

**Pull request merged**